### PR TITLE
Escpos Receipt printer

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
:warning: Work In Progress !!!

**Comment by BebZ** [comment]
Wahou ! So nice, a receipt printer soon availble ! What else... coffee maker soon ?
for your WIP, maiybe it can help ? http://mike.bitrevision.com/blog/what-is-escpos-and-how-do-i-use-it
it's very well made :)

**Comment by frederic34** [comment]
@BebZ esc/pos Library come from this guy
Printing is not the problem, it's to have a comprehensive template system

File: htdocs/core/modules/modReceiptPrinter.class.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        $this->db = $db ;
+        $this->numero = 67000;
+        // Family can be 'crm','financial','hr','projects','products','ecm','technic','other'
+        // It is used to group modules in module setup page
+        $this->family = "technic";
+        // Module label (no space allowed), used if translation string 'ModuleXXXName' not found (where XXX is value of numeric property 'numero' of module)
+        $this->name = preg_replace('/^mod/i','',get_class($this));
+        // Module description, used if translation string 'ModuleXXXDesc' not found (where XXX is value of numeric property 'numero' of module)
+        $this->description = "ReceiptPrinterDesc";
+        $this->version = 'dolibarr';    // 'development' or 'experimental' or 'dolibarr' or version
```
Comment (rdoursenaud) [inline htdocs/core/modules/modReceiptPrinter.class.php:54]: Since this is a WIP, I believe that version should be development!

**Comment by frederic34** [comment]
@rdoursenaud https://github.com/Dolibarr/dolibarr/pull/3889

**Pull request merged**